qa/L0_custom_flags/check_results.py

```diff
@@ -66,7 +66,7 @@ def check_perf_global(self):
 
             # In contents, search for "stabilizing with px latency"
             percentile = self._config["perf_analyzer_flags"]["percentile"]
-            token = f"Stabilizing using p{percentile} latency"
+            token = f"Stabilizing using p{percentile}latency"
```

Comment:
Will the removal of the space make it harder to read the logs?

Reply:
I didn't change the code, only the test. I could update the code, but I thought it's safer to fix the test to get it matching behavior (when the behavior is correct).